src/autoscaler/pruner/appmetricsdb.go

```diff
@@ -63,4 +34,5 @@ func (amdp *AppMetricsDbPruner) PruneOldMetrics() {
 		amdp.logger.Error("prune-appmetricsdb", err)
```

Comment:
This should say failed-prune-appmetrics. Look at Diego code to see how these messages should be formatted.